src/main/java/hudson/plugins/parameterizedtrigger/TriggerBuilder.java

```diff
@@ -181,12 +184,44 @@ public String getProjectListAsString(List<Job> projectList){
         return projectListString.toString();
     }
 
-
     @Override
     public Collection<? extends Action> getProjectActions(AbstractProject<?, ?> project) {
         return ImmutableList.of(new SubProjectsAction(project, configs));
     }
 
+    private boolean canDeclare(AbstractProject owner) {
+        // In Hudson 1.341+ builds will be triggered via DependencyGraph
+        // Inner class added in Hudson 1.341
+        String ownerClassName = owner.getClass().getName();
+        return DependencyGraph.class.getClasses().length > 0
```

Comment:
The plugin now requires newer Jenkins core version than in 2016, maybe this check is not required anymore